src/components/UI/Display/MAPI/clusters/GettingStarted/GettingStartedGetAccess.tsx

```diff
@@ -176,11 +148,11 @@ const GettingStartedGetAccess: React.FC<IGettingStartedGetAccessProps> = () => {
             role='presentation'
             aria-hidden={true}
           />{' '}
-          <code>--certificate-organizations</code> is a flag that sets what
-          group you belong to when authenticating against the Kubernetes API.
-          The default superadmin group on RBAC (Role Based Access Control)
-          enabled clusters is <code>system:masters</code> . All clusters on AWS
-          have RBAC enabled, some of our on-prem (KVM) clusters do not.
+          <code>--certificate-group</code> is a flag that sets what group you
+          belong to when authenticating against the Kubernetes API. The default
+          superadmin group on RBAC (Role Based Access Control) enabled clusters
+          is <code>system:masters</code> . All clusters on AWS have RBAC
+          enabled, some of our on-prem (KVM) clusters do not.
```

Comment:
Not sure if it's worth mentioning `KVM` here, as KVM users won't see this page anyway

Reply:
I'll remove the sentence 👍